sale_automatic_workflow/models/automatic_workflow_job.py

```diff
@@ -54,6 +54,25 @@ def _do_validate_sale_order(self, sale, domain_filter):
         sale.action_confirm()
         return "{} {} confirmed successfully".format(sale.display_name, sale)
 
+    def _do_send_order_confirmation_mail(self, sale):
+        """Send order confirmation mail, filter ensure no duplication.
+        Prevent mail send dupplication according to odoo
+        /addons/website_sale/models/sale_order.py#L357"""
```

Comment:
```suggestion
        """Send order confirmation mail, while filtering to avoid duplication if the message was already sent in odoo/addons/website_sale/models/sale_order.py#L357"""
```

Also, the reference is not very resilient to change, so maybe you could reference the actual method, model and module instead...

Reply:
I still don’t see this change… Did you push it?